.github/workflows/ci.yml

```diff
@@ -41,3 +41,9 @@ jobs:
           PGP_SIGNING_PASSWORD: ${{ secrets.PGP_SIGNING_PASSWORD }}
           SONATYPE_USERNAME: ${{ secrets.SONATYPE_USERNAME }}
           SONATYPE_PASSWORD: ${{ secrets.SONATYPE_PASSWORD }}
+      - name: Upload test report
+        if: always()
+        uses: actions/upload-artifact@v2
+        with:
+          name: my-artifact
```

Comment:
Maybe you have a more meaningful name for the artifact than `my-artifact`?

Reply:
Changed